src/axom/multimat/multimat.hpp

```diff
@@ -487,40 +511,123 @@ class MultiMat
   void transposeField_helper(int field_idx);
 
   template <typename T>
-  MapBaseType* helper_copyField(const MultiMat&, int map_i);
+  MapUniquePtr helper_copyField(const MultiMat&, int map_i);
+
+  /*!
+   * \brief Returns the associated cell set.
+   */
+  RangeSetType& getCellSet() { return m_sets[0]; }
+  const RangeSetType& getCellSet() const { return m_sets[0]; }
+
+  /*!
+   * \brief Returns the associated material set.
+   */
+  RangeSetType& getMatSet() { return m_sets[1]; }
+  const RangeSetType& getMatSet() const { return m_sets[1]; }
+
+  /*!
+   * \brief Returns a reference to the corresponding array of offsets for a
+   *        static relation corresponding to a layout.
+   *
+   * \param layout The layout type of the relation (cell- or mat-dominant)
+   */
+  std::vector<SetPosType>& getRelBeginVec(DataLayout layout);
```

Comment:
Minor: Should the return type be a type alias (or possibly an existing type from the relation?) so it'll be easier to switch it over to `axom::Array`?